javascript/javascript3/week2/hWExcercise.js

```diff
@@ -0,0 +1,101 @@
+//Movie excercise
+fetch("https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json")
+    .then(data => data.json())
+    .then(data => {
+        let badMovies = data.filter(item => item.rating<4)
+        console.log(badMovies)
+        return badMovies
+    })
+    .then(data=>{
+        let badMoviesSince2000 = data.filter(item => item.year > 2000)
+        console.log(badMoviesSince2000)
+        return badMoviesSince2000
+    })
+;
+
+
+//Promise that resolves after set time
+function resolveLater(resolveAfter){
+    return new Promise(resolve => {
+        setTimeout(resolve, resolveAfter * 1000)
+    })
+}
+
+resolveLater(8).then(()=> console.log("I am called asyncronously"));
+
+/* //With async/await
+async function resolveLater(resolveAfter){
+    try{
+        let waitingTime = await setTimeout(()=>{console.log(`Logged after ${resolveAfter} seconds`)}, resolveAfter * 1000);
+        return waitingTime;
+    } catch{
+        console.log("Error!")
+    }
+}
+
+*/
```

Comment:
Why is this commented out?

Reply:
It just does exactly the same as the previous function(lines 17-22).. I only did it with async-await as practice..